suites/reef/upgrades/tier1-6x-to-7x-downstream-upgrade.yaml

```diff
@@ -0,0 +1,182 @@
+#################################################################################
+# Automation support for upgrade from RHCS-6 to RHCS-7 in RHEL-9
+#
+#--------------------------------------------------------------------------------
+# Cluster Configuration: conf/reef/upgrades/1admin-3node-1client-upgrade.yaml
+#--------------------------------------------------------------------------------
+#
+# Test Steps:
+#--------------------------------------------------------------------------------
+# - Deploy RHCS-6 cluster in RHEL-9
+# - Upgrade cluster from RHCS-6 to RHCS-7
+# - Validate cluster status
+# - Run I/O's
+#--------------------------------------------------------------------------------
+#################################################################################
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                rhcs-version: 6.1
+                release: "latest"
```

Comment:
Remove below parameter 
                rhcs-version: 6.1
                release: "latest"
               
Add as below
custom_image: registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177
custom_repo:  http://download-node-02.eng.bos.redhat.com/rhel-9/rel-eng/RHCEPH-6/RHCEPH-6.1-RHEL-9-20230602.0

It help us to maintain, Not depend on recipe file

Reply:
Remove  release: "latest"